edenai_apis/apis/amazon/tests/test_amazon_helpers.py

```
@@ -0,0 +1,273 @@
+        assert result is not None
+        assert len(result.extracted_data) == 0
+
+
+# ---------- HTTP error wrapper tests ----------
+
+
+class TestHandleAmazonCall:
+    @pytest.mark.unit
+    @pytest.mark.amazon
+    def test_success_returns_response(self):
+        func = lambda **kwargs: {"result": "ok"}
```

Ruff (0.15.10)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `func` as a `def`

(E731)